app/views/application/_head.html.erb

```diff
@@ -15,7 +15,7 @@
   <link rel="apple-touch-icon" sizes="167x167" href="<%= asset_pack_url('media/images/govuk-apple-touch-icon-167x167.png') %>">
   <link rel="apple-touch-icon" sizes="152x152" href="<%= asset_pack_url('media/images/govuk-apple-touch-icon-152x152.png') %>">
   <link rel="apple-touch-icon" href="<%= asset_pack_url('media/images/govuk-apple-touch-icon.png') %>">
-
+  <script>L_DISABLE_3D = true;</script>
```

Comment:
script tags should not be placed in the header of the application, but in the body - just like the one we currently have in ```app/views/layouts/application.html.erb``` which would be the right place for this?

Reply:
[According to the documentation](https://leafletjs.com/reference-1.7.1.html#global-switches), it is important that this flag is set before leaflet is loaded. I thought that adding it before it is included in the packed JS might work, but it didn't, so now I'm making sure it is set before *any* js is loaded, just to make sure that that isn't the problem.